### PR TITLE
Feature: split publishers screen into visited and discover

### LINKED ISSUE #1850: [Feature] Split "Your ⚡️ Websites" and "Other ⚡️ Websites" into "Discover" (Other) and "Visited" (Yours)
### Feature description

- Split "Websites" screen into 2
     image

### Describe the solution

- update nav-items (tab and popup view)
- split content
- not sure yet how to handle the closable boxes section now (could be removed if all boxes closed i.e.? needs to be discussed maybe)

### PR DESCRIPTION
**Pull request by rolznz**
### Describe the changes you have made in this PR

Split publishers screen into visited and discover, added new discover links in the nav and popup, updated copy

### Link this PR to an issue [optional]

Fixes #1850

### Type of change

(Remove other not matching type)

- `feat`: New feature (non-breaking change which adds functionality)

### Screenshots of the changes [optional]

_Add screenshots to make your changes easier to understand. You can also add a video here._

Screenshot_430

Screenshot_428
Screenshot_429
image

### How has this been tested?

_Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration_

### Checklist

- [X] My code follows the style guidelines of this project and performed a self-review of my own code
- [X] New and existing tests pass locally with my changes
- [X] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

**Comment by rolznz** [comment]
Notes:
- Do we want to rename "Publishers" to "VistedSites"?

File: src/app/screens/Publishers/noPublishers.test.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import { render, screen } from "@testing-library/react";
```
Comment (qqqzhch) [inline src/app/screens/Publishers/noPublishers.test.tsx:1]: Are there two test files here now? The other is index.test.tsx?

Comment (escapedcat) [reply src/app/screens/Publishers/noPublishers.test.tsx:1]: Yes, looks like it:
https://github.com/getAlby/lightning-browser-extension/tree/e316da4de115eb542160170a836e0f204a069338/src/app/screens/Publishers

@rolznz I think you could just move that test into the test-index file. Or did you have any issues with that?

**Review by escapedcat — COMMENTED** [review]
ack

**Comment by escapedcat** [comment]
> * Do we want to rename "Publishers" to "VisitedSites", /publishers to /visited, etc?

I would support this. Wdyt @reneaaron @bumi ?

File: src/app/screens/Publishers/noPublishers.test.tsx (hunk 1 above)
Comment (rolznz) [reply src/app/screens/Publishers/noPublishers.test.tsx]: @escapedcat  @qqqzhch  I had some trouble with the mocks, but I found a solution. I have removed the test file I added and moved the test I made into the index test file

**Review by escapedcat — APPROVED** [review]

**Comment by reneaaron** [comment]
I think "websites" is the better term for this menu item. Publishers is not accurate (you can tip also non-publishers), "Visited" seems a bit odd to me. My vote goes for keeping this as "websites". (also the title on the page says `Your ⚡ websites`)

**Comment by rolznz** [comment]
> > * Do we want to rename "Publishers" to "VisitedSites", /publishers to /visited, etc?
> 
> I would support this. Wdyt @reneaaron @bumi ?

I have made a separate MR here: https://github.com/getAlby/lightning-browser-extension/pull/1914 (Note, it is branched off of this one)

**Comment by reneaaron** [comment]
Just tested this PR 👌 I like the changes, one thing with the new menu items: should we maybe remove the `settings`  from the main menu? You'll still be able to access it in the dropdown. 

I don't think you'll need the settings that often TBH.

image

**Review by reneaaron — APPROVED** [review]

**Pull request merged**